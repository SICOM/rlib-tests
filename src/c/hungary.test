#
# This test crashes RLIB 2.0.x because it uses:
#
# rlib_set_output_parameter(r, "pdf_fontname", "Liberation Mono");
#
# RLIB 2.0.x doesn't handle fonts that are not in the 14 base fonts.
#

COMMAND=./hungary
FORMATS="pdf"
EXPECTED_pdf=
EXPECTED_xml=
EXPECTED_txt=
EXPECTED_csv=
EXPECTED_html=
