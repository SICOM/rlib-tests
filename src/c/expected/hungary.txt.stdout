ÁRVÍZTÛRÕ TÜKÖRFÚRÓGÉP
árvíztûrõ tükörfúrógép
Page:   1
